RELEASE_NOTES.md

```diff
@@ -8,15 +8,7 @@
 [//]: # ( ### Deprecated)
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
-## Added
-- Validation of PublicKey in CSE module if it's used standalone.
-- 'isReady' flag to ComponentState because some components might require some initialization time even if all the inputs are valid.
-- 'isValid' now checks both 'isInputValid' and 'isReady' to be true.
-
-## Fixed
-- Handle Intent results if DropInActivity got destroyed.
-- Queue API request if DropInService is not yet bound to DropInActivity
-
-
-
-
+## Changed
+- Updated Material Components dependency to 1.3.0.
+- The `merchantAccount` parameter in `GooglePayConfiguration.Builder` is now optional.
+- Provided default `GooglePayConfiguration` and `CardConfiguration` in Drop-in.
```

Comment:
We also:
- Removed WeChatPay Component (but still have the WeChatPayActionComponent)
- Added Pix Component

Might as well leave the release notes up to date :)

Reply:
Hmm why though? I'm sure we missed more than these 2, so we can add them along with the remaining changes on release. I only added the Google Play ones here because I needed to make this quick fix and I thought I'll include the release notes anyway.